src/cascadia/TerminalApp/TerminalSettings.h

```diff
@@ -15,10 +15,14 @@ Author(s):
 #pragma once
 
 #include "TerminalSettings.g.h"
+#include "../TerminalSettingsModel/IInheritable.h"
 #include "../inc/cppwinrt_utils.h"
+#include "../../types/inc/colorTable.hpp"
 #include <DefaultSettings.h>
 #include <conattrs.hpp>
 
+using namespace Microsoft::Console::Utils;

```

Comment:
ooh can you put out a PR that deletes this? we should never put a `using` in a header.

/cc @carlos-zamora @zadjii-msft  :smile:

Reply:
In fact, only the cpp file needs to know about colorTable.hpp and the `using` block, correct?

Reply:
Ah sorry my bad, made a PR to fix it